src/components/shared/radio/radio.js

```diff
@@ -0,0 +1,44 @@
+import React from "react";
+import { hot } from "react-hot-loader";
+import "./radio.scss";
+import PropTypes from "prop-types";
+
+
+class Radio extends React.Component {
+    static propTypes = {
+        name: PropTypes.string,
+        id: PropTypes.string,
+        label: PropTypes.string,
+        additionalClassName: PropTypes.string,
+        defaultChecked: PropTypes.bool
+    };
+
+    constructor (props) {
+        super(props);
+        this.state = {};
+    }
+
+    handleRadioChange () {
+        console.log(`Selected property is ${this.props.id}`);
+    }
+
+    render () {
+        return (
+            <div className="formRadio">
+                <input
+                    className="formRadio__input"
+                    type="radio"
+                    name={this.props.name}
```

Comment:
to get rid of multiple this.props... you can destructure all the necessary props before `return`:
`const { name, id, .... } = this.props;`

Reply:
Refactored